### PR TITLE
New package: reddio-0.52

### PR DESCRIPTION
**Pull request by DerAndereJohannes**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Note
First time contributing here. Been using reddio for a while, I saw that #14223 is open and the work was basically already done but not submitted in #14281. I made minor changes and decided to submit. Let me know if I should make any changes. Thanks!

### CODE REVIEW

File: srcpkgs/reddio/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'reddio'
+pkgname=reddio
+version=0.52
+revision=1
+wrksrc="${pkgname}-v${version}"
+depends="curl jq"
+short_desc="Command-line interface for Reddit written in POSIX"
```
Comment (paper42) [inline srcpkgs/reddio/template:7]:
```suggestion
short_desc="Command-line interface for Reddit written in POSIX sh"
```

I am not sure this meets quality requirements if it's just a shell script.

File: srcpkgs/reddio/patches/change_share_path.patch [hunk 2]
```diff
@@ -0,0 +1,13 @@
+diff --git a/reddio b/reddio
+index dd65718..bc78105 100755
+--- a/reddio
++++ b/reddio
+@@ -197,7 +197,7 @@ main()
+ 	fi
+ 
+ 	# Library path environment variable has precedence over config
+-	lib_dir=${REDDIO_LIB:-${lib_dir:-/usr/local/share/reddio}}
++	lib_dir=${REDDIO_LIB:-${lib_dir:-/usr/share/reddio}}
```
Comment (paper42) [inline srcpkgs/reddio/patches/change_share_path.patch:10]: can we use `build_style=gnu-makefile` and set `make_install_args="LIBDIR=/usr/share/reddio"` instead?

File: srcpkgs/reddio/INSTALL.msg [hunk 3]
```diff
@@ -0,0 +1 @@
+Optional: netcat to authenticate with your Reddit account credentials
```
Comment (paper42) [inline srcpkgs/reddio/INSTALL.msg:1]: we don't really use INSTALL.msg for optional dependencies and this seems to be an important dependency. I would remove INSTALL.msg and include it in $depends.

**Comment by DerAndereJohannes** [comment]
@paper42 Thank you for your feedback! I put all the changes in that you proposed. The make file had a handy prefix variable to solve the install issue and remove the patch!

As to if the program meets the quality requirements: It is a very handy and sophisticated set of  programs to make user scripts far more powerful. I would have guessed the initial people proposing it think the same. On that note, if the team does not think it is sufficient enough, I will understand and we could get rid of this and #14223.

**Pull request closed without merging**